bin/src/Main.purs

```diff
@@ -528,7 +547,7 @@ main = do
             logInfo "Set up a new Spago project."
             logInfo "Try running `spago run`"
           Fetch args -> do
-            { env, fetchOpts } <- mkFetchEnv offline args
+            { env, fetchOpts } <- mkFetchEnv offline (Record.insert (Proxy :: _ "isRepl") false args)
```

Comment:
Any reason for the choice to use `insert` here vs. `merge` on line 562? Arbitrary?

Reply:
I tend to use `insert` if I'm only dealing with one key, could try to swap it with merge and see if that works (all the record things are quite finicky when it comes to envs, for some reason)

Reply:
I don't prefer either, it just looked like you were using `merge` with one field below and I didn't know if there was a reason to choose one or another that emerged from your env-wrangling :)